File: fh/fhCoreLite/basicControls/src/main/java/pl/fhframework/model/forms/RowDropdownItem.java
package pl.fhframework.model.forms;

import pl.fhframework.annotations.Control;

@Control(parents = {Dropdown.class, Repeater.class}, canBeDesigned = true)
public class RowDropdownItem extends DropdownItem implements Iconable {
    public RowDropdownItem(Form form) {
        super(form);
    }
}

File: fh/packages/components/basic-controls/source/controls/Dropdown/RowDropdownItem.ts
import { HTMLFormComponent } from 'fh-forms-handler';
import {DropdownItem} from "./DropdownItem";

class RowDropdownItem extends DropdownItem {
   constructor(componentObj: any, parent: HTMLFormComponent) {
        super(componentObj, parent);
    }
}

export {RowDropdownItem};
